code-challenges.test.js

```diff
@@ -65,3 +135,20 @@ const accountTransactions3 = []
 // Expected output: []
 
 // b) Create the function that makes the test pass.
+
+// Puesdo code: Create the function called arraySum.
+// Give the function the parameter 'array' to be used in the funciton. This will match the datatype of the argument.
+// Create a new variable in the funciton named sum and set it equal to 0 for the baseline.
+//Inside the function use the .map() method to iterate through the origin array.
+// inside .map name the value 'num' for carity.
+// after the arrow syntax place the variable sum += to num so each number in the array will be added to the sum.
+// return sum as the new value for each index.
+
+const arraySum = (array) => {
+  let sum = 0
+  let newArray = array.map((num) => {
+    sum += num
+    return sum
+  })
+  return newArray
+}
```

Comment:
Great job with your code challenges Spencer! You're is doing a good job of setting up a full-proof plan of attack and your code is clean! 🔥